contracts/platform/FundRequestContract.sol

```diff
@@ -105,12 +108,39 @@ contract FundRequestContract is Owned, ApproveAndCallFallBack {
         return true;
     }
 
+    function refund(bytes32 _platform, string _platformId, address _funder) public onlyCaller returns (bool) {
+        uint256 tokenCount = fundRepository.getFundedTokenCount(_platform, _platformId);
+        for (uint i = 0; i < tokenCount; i++) {
+            address token = fundRepository.getFundedTokensByIndex(_platform, _platformId, i);
+            uint256 tokenAmount = fundRepository.refundToken(_platform, _platformId, _funder, token);
+            if (tokenAmount > 0) {
+                if (token == ETHER_ADDRESS) {
+                    _funder.transfer(tokenAmount);
+                } else {
+                    require(ERC20(token).transfer(_funder, tokenAmount), "transfer of tokens from contract failed");
+                }
+            }
+            emit Refund(_funder, _platform, _platformId, token, tokenAmount);
+        }
+    }
+
+    function validRefundServerSignature(bytes32 platform, string platformId, bytes32 r, bytes32 s, uint8 v) internal view returns (bool) {
+        bytes32 h = keccak256(abi.encodePacked(createRefundMsg(platform, platformId)));
+        address signerAddress = ecrecover(h, v, r, s);
+        return claimSignerAddress == signerAddress;
```

Comment:
it is indeed the same address, but the name of the variable is not correct anymore

Reply:
@Qkyrie ^^

Reply:
That method isn't used anymore, forgot to remove it